README.md

```diff
@@ -1,2 +1,10 @@
-# Apple_Clone
-Microverse Solo Project
+# micro-apple
+
+This project is to learn how to apply backgrounds and gradients with CSS3.
```

Comment:
Please make this README more descriptive. This will help other developers understand how to get started with the project. It is required for this project's completion. Look at this [post](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)